doc/Analytics.xml

```diff
@@ -4368,6 +4368,84 @@ xmlns:acme="http://www.acme.com/schema"&gt;
     </tt:ParentTopic>
   </tt:Messages>
 </tt:RuleDescription>
+]]></programlisting>
+    </section>
+    <section>
+      <title>Object Track Aggregation</title>
+      <para>The Object Track Aggregation rule generates an object track aggregation event for below
+        listed scenarios </para>
+      <itemizedlist>
+        <listitem>
+          <para> Optionally, Initial aggregation after an object appears OR an Intermediate aggregation in the field of view 
+            or an Intermediate aggregation while the object is present in the field of view.</para>
+        </listitem>
+        <listitem>
+          <para> Final aggregation when the object is no longer visible in the field of view. </para>
+        </listitem>
+      </itemizedlist>
+      <variablelist role="op">
+        <varlistentry>
+          <term>Parameters</term>
+          <listitem>
+            <para role="param">ClassFilter [tt:StringList]</para>
+            <para role="text">List of classes to be aggregated.</para>
+            <para role="param">ConfidenceLevel - optional [xs:float]</para>
+            <para role="text">Minimum confidence level of object classification for generating
+              aggregation event. </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Topic</term>
+          <listitem>
+            <para role="param">tns1:RuleEngine/ObjectTrack/Aggregation</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Source</term>
+          <listitem>
+            <para role="text">See <xref linkend="_msgSource"/></para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Data</term>
+          <listitem>
+            <para role="param">AggregationStart - mandatory [xs:dateTime] </para>
+            <para role="text">All aggregation events shall set object track start time as the
+              AggregationStart time. </para>
+            <para role="param">AggregationEnd - mandatory [xs:dateTime]</para>
+            <para role="text">AggregationEnd time is usually set to the the last time the object was
+              observed before the aggregation. Final aggregation event shall set the field as the
+              object track end time.</para>          
+            <para role="param">ObjectTrackAggregation - mandatory [tt:Object]</para>
```

Comment:
Suggest to use already defined name `Object` as in A.1.1

Reply:
OK to change the field name as "Object", but it will be ambiguous to refer to A.1.1.1 as that section specifies tt:Object as unbounded parameter while our current proposal is expecting only one instance of aggregated object data.

Reply:
Agree with not referring to A.1.1.1 to avoid confusion.